aws/data_source_aws_ebs_default_kms_key.go

```diff
@@ -0,0 +1,35 @@
+package aws
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceAwsEbsDefaultKmsKey() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsEbsDefaultKmsKeyRead,
+
+		Schema: map[string]*schema.Schema{
+			"key_id": {
```

Comment:
The `aws_ebs_default_kms_key` resource that has been merged for release uses the `key_arn` argument so this should be updated to match. 👍 

```suggestion
			"key_arn": {
```